climada/hazard/tc_tracks.py

```diff
@@ -956,10 +958,16 @@ def from_simulations_storm(cls, path, years=None):
         # a bug in the data causes some storm tracks to be double-listed:
         tracks_df = tracks_df.drop_duplicates(subset=["year", "tc_num", "time_delta"])
 
-        # conversion of units and time
+        # conversion of units
         tracks_df['rmw'] *= (1 * ureg.kilometer).to(ureg.nautical_mile).magnitude
         tracks_df['wind'] *= (1 * ureg.meter / ureg.second).to(ureg.knot).magnitude
+
+        # convert from 10-minute to 1-minute sustained winds
+        tracks_df['wind'] /= 0.88
```

Comment:
This was a long-standing issue: All read functions in CLIMADA make sure that track wind speeds are 1-minute sustained winds, except this one. So, I fixed it.

Reply:
Great! Could you just comment year why the number `0.88`, I do not get it right away.

Reply:
As this is part of the `read_simulations_storm` function, I just took the value from the STORM paper Bloemendaal et al. 2020 that is cited in the function docstring. With my latest commit, this is also mentioned in the inline comment. It's also a very common factor for this kind of conversion, see the docstring to `IBTRACS_AGENCY_1MIN_WIND_FACTOR`.

Reply:
Thanks! Would it then make sense to pack it into a constant?

Reply:
Yes, why not. Added in 9347bd30